pyccl/core.py

```diff
@@ -871,6 +871,44 @@ def compute_sigma(self):
         status = lib.cosmology_compute_sigma(self.cosmo, pk.psp, status)
         check(status, self)
 
+    def get_linear_power(self, name=None):
+        """Get the :class:`~pyccl.pk2d.Pk2D` object associated with
+        the linear power spectrum with name `name`.
+
+        Args:
+            name (:obj:`str` or `None`): name of the power spectrum to
+                return. If `None`, `'delta_matter_x_delta_matter'` will
+                be used.
+
+        Returns:
+            :class:`~pyccl.pk2d.Pk2D` object containing the linear
+            power spectrum with name `name`.
+        """
+        if name is None:
+            name = 'delta_matter_x_delta_matter'
```

Comment:
You should make this the default in the function signature.

Reply:
meaning `name='delta_matter_x_delta_matter'` as default value?

Reply:
yup.

Reply:
ok, done